CONTRIBUTING.md

```diff
@@ -348,6 +348,27 @@ You can visit <https://github.com/endoflife-date/endoflife.date/new/master/produ
 
 ## ✅ Validating your changes
 
+If you're using an IDE like `vscode` or `vim` (or any other IDE that support jsonschema validation),
+you can use [this jsonschema](./products/schema.json) to validate the new product.
```

Comment:
Wouldn't it be better to move the schema file to the repository root with the name `product-schema.json` ?

Reply:
We can do that, no problem :)